common/libs/validators.js

```diff
@@ -139,3 +139,7 @@ export function isValidRawTx(rawTx: RawTransaction): boolean {
 
   return true;
 }
+
+export function isValidPath(dPath: string) {
+  return dPath.split('\'/').length === 4;
```

Comment:
maybe a comment here explaining what's going on. I get that the deterministic path is being split, but its not extremely clear why 4 is important

Reply:
https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki has more information, and test cases, I'll try to add a useful comment and implement some of its test cases.